src/main/java/org/commcare/formplayer/application/WebAppContext.java

```diff
@@ -107,11 +107,6 @@
     @Value("${detailed_tagging.tag_names:}")
     private List<String> detailedTagNames;
 
-    @Override
-    public void configureDefaultServletHandling(DefaultServletHandlerConfigurer configurer) {
-        configurer.enable();
-    }
```

Comment:
this is only needed for serving static dependencies and errors in the new version version